__tests__/util/itinerary.js

```diff
@@ -1,4 +1,6 @@
-import {isTransit, routeComparator} from '../../lib/util/itinerary'
+import coreUtils from '@opentripplanner/core-utils'
```

Comment:
Re: import style, it'd be good to have an answer on whether the style influences build size. If so, let's use the more precise import style (that refs the `lib/` directory).